student.rb

```diff
@@ -1,12 +1,16 @@
 require './person'
-
 class Student < Person
```

Comment:
- [ ] **Required:** The student's classroom should be readable. Consider the snippet below.

```
attr_reader :classroom
```